packages/polar/sample-project/scripts/sample-script.js

```diff
@@ -1,18 +1,17 @@
-const { Contract, createAccounts } = require("secret-polar");
-
+const { Contract, getAccountByName } = require("secret-polar");
 
 async function run (runtimeEnv) {
+  const contract_owner = getAccountByName("account_0", runtimeEnv);
   const contract = new Contract('sample-project', runtimeEnv);
   await contract.parseSchema();
 
-  const account = runtimeEnv.config.networks.testnet.accounts[0];
-  const deploy_response = await contract.deploy(account);
+  const deploy_response = await contract.deploy(contract_owner.account);
   console.log(deploy_response);
 
-  const contract_info = await contract.instantiate({"count": 102}, "deploy test", account);
+  const contract_info = await contract.instantiate({"count": 102}, "deploy test", contract_owner.account);
```

Comment:
We can use type `(Account | UserAccount)` here and then only pass `contract_owner`.